mitmproxy/io/proto/http.proto

```diff
@@ -0,0 +1,93 @@
+syntax='proto2';
+
+message HTTPFlow {
+    optional HTTPRequest request = 1;
+    optional HTTPResponse response = 2;
+    optional HTTPError error = 3;
+    optional ClientConnection client_conn = 4;
+    optional ServerConnection server_conn = 5;
+    optional bool intercepted = 6;
+    optional bool marked = 7;
+    optional string mode = 8;
```

Comment:
Next step: intercepted, marked and mode should all be items in a general annotation mechanism in the format. That is, this should be in a separate annotations table with a nice general interface.

Reply:
DB wrapper just served as stub code to test protobuf to/from file, so table schemas did not receive much attention (yet). I will add for sure those fields in annotation table, but do you think that should be reflected in the proto format, too?